test/cypress/integration/group-paying.spec.js

```diff
@@ -161,7 +161,7 @@ describe('Group Payments', () => {
     cy.getByDT('modal').within(() => {
       cy.getByDT('payRecord').find('tbody').children().should('have.length', 1)
       cy.getByDT('payRow').eq(0).find('input[data-test="amount"]').should('have.value', '178.57')
-      cy.getByDT('payRow').eq(0).find('input[data-test="amount"]').clear({ force: true }).type('100')
+      cy.getByDT('payRow').eq(0).find('input[data-test="amount"]').clear({ force: true }).type('100', { force: true })
```

Comment:
Why is `force: true` necessary?

Reply:
The latest version of Cypress would get to this part and give an error because there's a div or something showing a $ sign over an input field, and even though it doesn't prevent input from the user, Cypress thinks it does, so it fails (unless this is passed in).